conf/app.routes

```diff
@@ -648,6 +648,7 @@ GET        /status/processing                      controllers.TrustStatusContro
 GET        /status/not-found                       controllers.TrustStatusController.notFound()
 GET        /status/down                            controllers.TrustStatusController.down()
 GET        /status/locked                          controllers.TrustStatusController.locked()
+GET        /status/claimed                         controllers.TrustStatusController.claimed()
```

Comment:
Can this be already-claimed rather than claimed?

This will help us with monitoring the endpoints being visited in Kibana. We already have a /claimed within claim-a-trust-frontend.